esmvaltool/_task.py

```diff
@@ -392,8 +393,12 @@ def _control_ncl_execution(self, process, lines):
 
     def _start_diagnostic_script(self, cmd, env):
         """Start the diagnostic script."""
+        logger.debug("**************************************")
         logger.info("Running command %s", cmd)
-        logger.debug("in environment\n%s", pprint.pformat(env))
+        logger.debug("In environment:")
+        for index, value in env.items():
+            logger.debug("env: %s:%s", index, value)
```

Comment:
This is a change I just made.

How useful is it to print the env dictionairy in `_task.py`? I have never looked at it and it's not really the kind of debug statement that I want to see. Personally, I would remove it as its long and confusing. However, if you think it's useful, then maybe cleaning up the way its printed might be useful

Reply:
careful with that, `env` is not really a dictionary unless you make it one, it's a system dict but not a Python dict